### DISCUSSION TITLE
Directory Tree

### DISCUSSION THREAD

**Discussion by offmybach** [comfyanonymous/ComfyUI#1813] [Q&A]
My Comfyui portable folder is a bloody mess. Not all of the files were installed in their proper location and self-contained. I have random .py files all over the place and too many folders without proper homes. Can someone kindly share their heirarchy of files for their portable folder with the accompanying .py and readme etc. files included. I desperately need one organized folder. I'm not an expert here but have been spending all of my spare time attempting to learn. Thanks in advance.

**Comment by phineas-pta** [comment]
https://github.com/comfyanonymous/ComfyUI/tree/master
